tools/props/Versions.props

```diff
@@ -58,5 +58,7 @@
     <SystemDataOdbcVersion>4.6.0</SystemDataOdbcVersion>
     <SystemNetSocketsVersion>4.3.0</SystemNetSocketsVersion>
     <XunitVersion>2.4.1</XunitVersion>
+    <MicrosoftDotNetRemoteExecutorVersion>5.0.0-beta.20206.4</MicrosoftDotNetRemoteExecutorVersion>
```

Comment:
Should we use other stable versions instead of the beta version in the project?

Reply:
There isn't a stable version yet, so I think it's fine since it's only used for tests.

Reply:
There is no stable version. You can take a look [here](https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&view=versions&package=Microsoft.DotNet.RemoteExecutor&version=5.0.0-beta.20181.2&protocolType=NuGet)